packages/create-bison-app/template/cypress/plugins/index.ts

```diff
@@ -68,11 +41,11 @@ export default (on, _config) => {
     disconnectDB: () => {
       return disconnect();
     },
-    factory: ({ name, attrs }) => {
-      const Factory = Factories[`${name}Factory`];
+    factory: ({ name, attrs }: { name: string, attrs: any }) => {
+      const Factory = (Factories as any)[`${name}Factory`];
       return Factory.create(attrs);
     },
-    login: (attrs: Pick<User, 'email' | 'password'>): Promise<LoginTaskObject> => {
+    login: async (attrs: Pick<User, 'email' | 'password'>): Promise<LoginTaskObject> => {
```

Comment:
Is there a specific reason for changing this to `async`? I suppose it's helpful for making it clear the function returns a promise, but the return type also does that. 

Also, if you wanted to, since this is async the promise chain at the end could be replaced with an `await`. Absolutely not a necessary change, of course.